Source/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "manifest_version": 2,
-  "name": "Carbon Footprint for Google Maps™",
+  "name": "TEST",
```

Comment:
Please don't include changes in your pull request that are not essential to your pull request.